src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaErrorDecoder.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaExceptionMessage;
+
+@Service
+public class CamundaErrorDecoder {
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Probably worth at some point wiring in object mappers as Spring configuration and injecting them where needed.  That way any additional configuration can be kept in one place.